docs/team.rst

```diff
@@ -0,0 +1,112 @@
+Astroquery Team
+===============
+
+Astroquery is primarily maintained by Adam Ginsburg and Brigitta Sipocz.
+
+Individual modules are partially maintained by their primary contributors
+or others who have taken over the responsibility:
```

Comment:
highlight that contriibutions for all modules are more than welcome (e.g maintaining doesn't mean that these people are responsible for fixing all the bugs and feature requests just that they are the one knowing the modules).

Reply:
:+1: and/or make that a suggestion here